src/CalendarUtils.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
 import moment from 'moment';
 
+import { getWeekdays } from './settings/utils/time';
+
 class CalendarUtils extends React.Component {
   static convertNewPeriodToValidBackendPeriod(period, event) {
-    const weekDays = ['SUNDAY', 'MONDAY', 'TUESDAY', 'WEDNESDAY', 'THURSDAY', 'FRIDAY', 'SATURDAY'];
+    const weekdays = getWeekdays();
```

Comment:
Could you please check possible to rename it to daysOfWeek ?

Reply:
Done